### DISCUSSION TITLE
Error: Cannot call write after a stream was destroyed

### DISCUSSION THREAD

**Discussion by PatrickIl** [prisma/prisma#2066] [Q&A] [answered]
I'm using Javascript. Any Prisma2 example I'm trying, I am getting the ERROR: "Cannot call write after a stream was destroyed", npx prisma generate. With both, Postgres and SQlite. 
I have just tried the HELLO-PRISMA example in the quickstart (start from scratch) and below's the error message I'm getting. Any hints? Am I missing something? Thanks for your help.
npm install @prisma/client

> @prisma/client@2.0.0-beta.1 postinstall /Users/patrickilunga/4.0 COURSES/GRAPHQL/ANDREWMEAD/PRISMA2/hello-prisma/node_modules/@prisma/client
> **node scripts/postinstall.js**

Error: Cannot call write after a stream was destroyed
npm WARN hello-prisma@1.0.0 No description
npm WARN hello-prisma@1.0.0 No repository field.

+ @prisma/client@2.0.0-beta.1
updated 1 package and audited 2 packages in 1.863s
found 0 vulnerabilities

**Comment by pantharshit00** [comment, marked as answer]
Hi @PatrickIl,

Can you please share which Operating System are you using?

**Reply by PatrickIl** [reply]
Hi @pantharshit00,l
Thank you for your assistance. I'm on **MacOS**. I managed to solve this problem with the help of the Slack Community. Some of the folders in the path of my project had spaces, and apparently Prisma2 does not accept path with spaces. **The problem is now solved**.
Thanks

**Reply by pantharshit00** [reply]
Oh nice, we are tracking the spaces problem here so you can subscribe to that if you are interested: https://github.com/prisma/prisma/issues/1973

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
